src/fields.js

```diff
@@ -39,7 +39,7 @@ const fields = [
   { label: "Spatial Query", field: "producedBy_samplingSite_location_rpt", type: "spatialquery" },
 ];
 
-// the initial camera position
+// the initial camera position, moorea
 const initialCamera = {
   facet: "List",
   longitude: -149.8169236266867,
```

Comment:
This is another magic number -- I'd recommend extracting out the latitude and longitude into symbolic constants like you've done elsewhere.